app/jobs/runtime/buildpack_installer.rb

```diff
@@ -14,12 +14,12 @@ def perform
           logger = Steno.logger('cc.background')
           logger.info "Installing buildpack #{name}"
 
-          buildpack = Buildpack.find(name: name)
+          buildpack = find_existing_buildpack
           if buildpack.nil?
             buildpacks_lock = Locking[name: 'buildpacks']
             buildpacks_lock.db.transaction do
               buildpacks_lock.lock!
-              buildpack = Buildpack.create(name: name)
+              buildpack = Buildpack.create(name: name, stack: 'unknown')
```

Comment:
Looks like stack already defaults to `'unknown'`. If you want to be explicit here, please pull out a constant.

Reply:
I just noticed that the default is defined on the controller rather than the model. Either way, we shouldn't have the same string in multiple places.

Reply:
@Gerg yeah having the defaults on the controller did seem odd, but `position`, `enabled` and `locked` are all defaulted here as well and it doesn't appear any sort of defaulting is done in the model.

We're not very familiar with Sequel, so if you can recommend a better practice, we would be happy to incorporate.